### PR TITLE
feat/added setup for the backend

### PR DESCRIPTION
**Pull request by DaleOrders**
Added structure for backend including nestjs, prisma and postgres dependencies.

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  "dependencies": {
+    "@nestjs/common": "^9.0.0",
+    "@nestjs/config": "^2.3.2",
+    "@nestjs/core": "^9.0.0",
+    "@nestjs/platform-express": "^9.0.0",
+    "@nestjs/typeorm": "^9.0.1",
+    "pg": "^8.11.0",
+    "reflect-metadata": "^0.1.13",
+    "rxjs": "^7.2.0",
+    "typeorm": "^0.3.16"
```
Comment (0916dhkim) [inline server/package.json:31]: Are these packages from NestJS template?

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    "@types/node": "18.16.12",
+    "@types/supertest": "^2.0.11",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^8.0.1",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "jest": "29.5.0",
+    "prettier": "^2.3.2",
+    "prisma": "^4.15.0",
```
Comment (0916dhkim) [inline server/package.json:48]: Prisma should be installed as dependency (not dev dependency)
Reference: https://www.prisma.io/docs/getting-started/quickstart

**Review by 0916dhkim — COMMENTED** [review]
That was fast! Just a couple minor comments below:

File: server/package.json (hunk 1 above)
Comment (DaleOrders) [reply server/package.json:31]: PG is the postgressql client and typeorm is an orm for typescript. Others were already included in nestjs package.

File: server/package.json (hunk 2 above)
Comment (DaleOrders) [reply server/package.json:48]: Prisma CLI is install as dev dependency. Prisma client is a dependency. Will install that as well.

Comment (DaleOrders) [reply server/package.json:49]: Done

Comment (DaleOrders) [reply server/package.json:49]: Done :)

**Pull request merged**